docs_src/examples/Ch-ExamplesModbusdatatypeconversion.md

```diff
@@ -20,28 +20,33 @@ to the `valueType` defined in the `properties` of the device resources.
 The following extract from a device profile defines the `rawType` as
 Int16 and the `valueType` as Float32:
 
-``` yaml
-deviceResources:
-  - name: "humidity"
-    description: "The response value is the result of the original value multiplied by 100."
-    attributes:
-      { primaryTable: "HOLDING_REGISTERS", startingAddress: "1", rawType: "Int16" }
-    properties:
-      valueType: "Float32"
-      readWrite: "R"
-      scale: "0.01"
-      units: "%RH"
-
-  - name: "temperature"
-    description: "The response value is the result of the original value multiplied by 100."
-    attributes:
-      { primaryTable: "HOLDING_REGISTERS", startingAddress: "2", rawType: "Int16" }
-    properties:
-      valueType: "Float32"
-      readWrite: "R"
-      scale: "0.01"
-      units: "degrees Celsius"
-```
+!!! edgey "EdgeX 2.0"
+    For EdgeX 2.0 the device profile has many changes. Please see [Device Profile](../microservices/device/profile/Ch-DeviceProfile.md) section for more details.
+
+!!! example "Example - Device Profile"
+
```

Comment:
I don't think this will render properly with this blank line here. Please remove or verify it renders properly.

Reply:
I verified it works with the following docker command:
```
docker run --rm -it -p 8000:8000 -v ${PWD}:/docs squidfunk/mkdocs-material:5.1.0
```

If I remove the blank line, the README format will break like this page https://github.com/edgexfoundry/edgex-docs/blob/master/docs_src/microservices/application/V2Migration.md

Reply:
Ok, Thansk!